generateElFF.py

```diff
@@ -95,7 +97,7 @@
         print(">>> loading tip density from ",options.tip_dens,"...")
         rho_tip, lvec_tip, nDim_tip, head_tip = GU.loadXSF( options.tip_dens )
```

Comment:
What if tip is calcualated by aims or cp2k? Some more general loading would be better, at least in the future.

Reply:
Yes, but keep this for future (I do not have time to test it now)